modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GoServerCodegen.java

```diff
@@ -322,34 +324,79 @@ public ModelsMap postProcessModels(ModelsMap objs) {
 
     @Override
     public OperationsMap postProcessOperationsWithModels(OperationsMap objs, List<ModelMap> allModels) {
-        objs = super.postProcessOperationsWithModels(objs, allModels);
+        // TODO: refactor abstractGoCodegen, decouple go client only code and remove this
         OperationMap objectMap = objs.getOperations();
         List<CodegenOperation> operations = objectMap.getOperation();
 
+        for (CodegenOperation operation : operations) {
+            // http method verb conversion (e.g. PUT => Put)
+            operation.httpMethod = camelize(operation.httpMethod.toLowerCase(Locale.ROOT));
+        }
+
+        // remove model imports to avoid error
         List<Map<String, String>> imports = objs.getImports();
         if (imports == null)
             return objs;
 
-        // override imports to only include packages for interface parameters
-        imports.clear();
+        System.out.println(imports);
+
+        Iterator<Map<String, String>> iterator = imports.iterator();
+        while (iterator.hasNext()) {
+            String _import = iterator.next().get("import");
+            if (_import.startsWith(apiPackage))
+                iterator.remove();
+        }
 
         boolean addedTimeImport = false;
         boolean addedOSImport = false;
+        boolean addedReflectImport = false;
         for (CodegenOperation operation : operations) {
             for (CodegenParameter param : operation.allParams) {
                 // import "os" if the operation uses files
-                if (!addedOSImport && ("*os.File".equals(param.dataType) || ("[]*os.File".equals(param.dataType)))) {
```

Comment:
Why this change? Would this break if an endpoint takes in a list/slice/array of Files?